csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DotNetCliInvoker.cs

```diff
@@ -31,6 +31,7 @@ private ProcessStartInfo MakeDotnetStartInfo(string args)
             };
             // Set the .NET CLI language to English to avoid localized output.
             startInfo.EnvironmentVariables["DOTNET_CLI_UI_LANGUAGE"] = "en";
+            startInfo.EnvironmentVariables["MSBUILDDISABLENODEREUSE"] = "1";
```

Comment:
Makes sense. There are some other variables we set for the "autobuilder". Some of them may be useful here too. Especially `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` which avoids a blocking keychain pop-up in cases on macos where `dotnet` has never been run before.

https://github.com/tamasvajk/codeql/blob/4608550e4814520390c960213ff5d21fcb7de3c9/csharp/autobuilder/Semmle.Autobuild.CSharp/DotNetRule.cs#L96

Reply:
Thanks, added as well.